src/main/java/roger/tasks/TaskList.java

```diff
@@ -0,0 +1,169 @@
+package roger.tasks;
+
+import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import roger.storage.StorageParser;
+
+
+/**
+ * Encapsulates the list of tasks in a Roger program.
+ */
+public class TaskList {
+    private List<Task> tasks;
+
+    /**
+     * Create an empty TaskList.
+     */
+    public TaskList() {
+        this.tasks = new ArrayList<>();
+    }
+
+    /**
+     * Create a TaskList, and initialise it with a list of tasks.
```

Comment:
I think US spelling will be more appropriate, as it caters to a wider audience. Hence I think it should be initialize instead of initialise.